### PR TITLE
Require Laravel 11

### PR DESCRIPTION
**Pull request by ttrig**
- [x] Test in production
- [x] https://github.com/glesys/butler-audit/pull/31
- [x] https://github.com/glesys/butler-auth/pull/36
- [x] https://github.com/glesys/butler-health/pull/29
- [x] https://github.com/glesys/butler-graphql/pull/67

### CODE REVIEW

File: src/config/app.php [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-    | This value is the name of your application. This value is used when the
+    | This value is the name of your application, which will be used when the
     | framework needs to place the application's name in a notification or
-    | any other location as required by the application or its packages.
+    | other UI elements where an application name needs to be displayed.
     |
     */
 
-    'name' => env('APP_NAME', 'butler-service'),
+    'name' => env('APP_NAME', 'Laravel'),
```
Comment (olvert) [inline src/config/app.php:16]: Why? 🙂

Comment (ttrig) [reply src/config/app.php:16]: Dunno :shrug: i'll change it back.

File: tests/laravel/.gitkeep
Comment (brother) [inline tests/laravel/.gitkeep:1]: directory will not be created via other means?

**Review by brother — APPROVED** [review]

File: tests/laravel/.gitkeep
Comment (ttrig) [reply tests/laravel/.gitkeep:1]: gone :+1:

**Review by olvert — APPROVED** [review]

**Review by olyckne — APPROVED** [review]

**Pull request merged**